communities/limassol-cy/community-assesment.md

```diff
@@ -0,0 +1,50 @@
+## Assessing Your Community's Needs
+
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - High School Hackers in and around Limassol, Cyprus
+  
+### Size
+- Currently there's around 50 people in the community.
+- We'd like to see around 100 people until July. 
+- We had a sharp increase when we founded the community last April, and after the last event, in December 2016.
```

Comment:
What are you doing in the event to help raise awareness that this is a longer-term community, and what that community offers?